### DISCUSSION TITLE
Activating the LightBulb UI with Hotkey/Shortcut/Function?

### DISCUSSION THREAD

**Discussion by ZGorlock** [Tyrrrz/LightBulb#322] [Q&A] [answered]
Hello, I have been trying to figure out how to activate the UI window without using the tray icon. I have the tray icon collapsed and I manually tweak the color temperature throughout the day.

I am mainly trying to have an icon pinned to the taskbar that, when clicked, pulls up the main window the same as if you had clicked the tray icon. I have a shortcut to LightBulb.exe but this will only start the program in the background (if it was not already running), and does nothing if it is already running.

Then I tried activating the window using an AutoHotkey script (which I would then turn into a shortcut for the taskbar), which did actually bring it up! However while it is "visible" and "enabled" it is still frozen and does not accept any input until the tray icon is clicked and it is activated in the normal way.

Do you know of a simpler way to create a shortcut that would accomplish this?

Or if not could this potentially be added in a future update?
Maybe by running "LightBulb.exe --show" or something similar.

**Comment by Tyrrrz** [comment, marked as answer]
Yeah I will add a shortcut

**Reply by Montell2099** [reply]
I apologize if it's inappropriate to necro an old thread in this way, but I am really enjoying this suggested implementation of using a hotkey to display the LightBulb UI! However, I would appreciate if it would work as a toggle to also be able to minimize the UI to the system tray without necessarily having to reach for my mouse and clicking "_Hide LightBulb to the system tray_" every time.

I've tried using **Win+M**, **Win+D**, and **Win+↓** without any luck. LightBulb gets minimized alright, though it is towards the taskbar & not to the system tray unfortunately. Would this be feasible or are there any potential issues I'm overlooking?

**Reply by Tyrrrz** [reply]
I will update it in the next release @Montell2099
